src/mlpack/methods/ann/ffn.hpp

```diff
@@ -215,6 +217,15 @@ class FFN
   void Forward(arma::mat&& input);
 
   /**
+   * Prepare the network for the given training data.
+   * This function won't actually trigger training process.
+   *
+   * @param predictors Input training variables.
+   * @param responses Outputs results from input training variables.
+   */
+  void SetTrainingData(const arma::mat& predictors, const arma::mat& responses);
```

Comment:
Do we have to provide this function? I guess it's only used internally, I'd like to keep the function set minimal, so maybe we can delete the function.